hs-abci-sdk/src/Tendermint/SDK/Modules/Validators/Types.hs

```diff
@@ -49,7 +51,19 @@ instance RawKey PubKey_ where
 
 
 instance A.ToJSON PubKey_
+instance A.ToJSONKey PubKey_
 instance A.FromJSON PubKey_
+instance A.FromJSONKey PubKey_
+
+instance HasCodec (Map PubKey_ Word64) where
+  encode = toStrict . A.encode
```

Comment:
While we’re at it, let’s replace `toStrict` with `Data.String.Conversions.cs`

Reply:
:+1: good call.